### PR TITLE
Fixed #2, train and test a classification model on vehicles dataset

### LINKED ISSUE #2: [Outreachy applications] Startup task: Train and test a classification model
This is a good way to get started with the environment and the problem domain. It will also provide the basis for a test case for future work. At a minimum, you should:

- load a dataset from the repo
- train a classification model from [scikit-learn](https://scikit-learn.org/stable/supervised_learning.html)
- compute an [evaluation metric](https://scikit-learn.org/stable/modules/model_evaluation.html) on a held-out test set

Feel free to include any additional steps you feel are relevant or you are interested in trying out, such as:

- basic exploratory analysis of the dataset
- data preprocessing
- hyperparameter tuning

When you start work on this task, please post a comment here indicating which dataset and model you will be working with so that other contributors can avoid duplicating your work.

### PR DESCRIPTION
**Pull request by Bolaji61**

### CODE REVIEW

File: dev/Omobolaji/vehicles.ipynb [hunk 1]
```diff
@@ -0,0 +1,958 @@
...
...JjzIpzymzHw+//s4Xz7cEx93FzILigFwd9by2t0daBDgRWpeEQ999iddGgfjY3dMRuYuW8/HEx8gxN+bEXOX0rN1BOF1apXlmTqsT9nrZX/u4dj5FAA6NmvA8pdNN9k5BUUMmrGELi2sb4L/LUPu6MdD997NS6+9dV32B+ay+i2Gj0f3JsTHgxEfr6Fn8zDCg33L8pzLyOXzLYf58rH++Li7kplvOn4dG4eyfMIdAOQUljDovZV0iahtf1BCg8ugsRR/8RoyNxO38XPRH92NTLtgnc/FDecut2OIP2G1WGYmU/zfqfbHYcFgNDL3h818PGEoIX5ejFjwHT1bNSa8dmBZnqn39ix7veyv/Ry7kGa1jYWr/qF9RF2HxhXdtxsNG9end6fBRLVvxewF07h3wCNWeQryCxnUa3jZ+183fsPaVaYHONNmTWTF97/z8/e/06VHR6a8/DRTnnzZrpgMRiNzf97Ox0/caTpPvfczPSMbEh5afp6aOrhr2etlW2M5lpBe9v6R6DYU6/T8uOOoXXFYxySZt+koH93TnhAvN0Ys+4eejYMID/SyyjegaSgv9rrFalnHegF8P7ILADnFOu7+Yiu3NgjEkQxGI3O+Xs3iySMJCfDhodmfEh3VjPC6QWV5PvltKwM6RjKsVwdOJ6Tx1HvfsnrBsw6PY+6PW/h4/N2mz/k7P9CzZSPCQwPK8kwdWl7pXrblYNnnPMjHk6UT78XFSUthSSn3zvuOni0bEezraX9gQuDSazglP7+HzM/Cbfg0DHEHkZlJVtn0J3ZXWXF17nI3xgsn7Y/FQt3ebfBpFMov3SdTq104neeOZvWgmTb5Lqzfy/Ev1jNkm/W5vdWzgzn7205O/G8jvk3q0Oerqfx866SrC0Kjoe7scZwZ+TK65AwiVr5D7vqdlJw6X5YlYFh/DDn5HI9+At9BPaj94mjin5oPQOm5ZE7eYfs5ytu4i4ylv9Ns8+Kri6eS+PynPkva01MxpKYR/OVHFG39G/2Z8kqXISWFrNfexGuEdSW6ZM9+Ukc9DoDw8ab2j19RsnO3ffEACA1uD0yg8IOXkNnpeLzwPvqDO02VVzNdzGZ0W/8AQNuqM673PkbRwpdxatcDnJwpfONJcHbF85XF6GI2IzPtfyiIELjc8SjFX81B5mbg9tgb6I/vQaYlWOdzccO580AMFp9nw6HtGA6Z
```
Comment (dzeber) [inline dev/Omobolaji/vehicles.ipynb]: Did you mean (15, 15)?

Comment (dzeber) [inline dev/Omobolaji/vehicles.ipynb]: Also, please add a comment describing how you decided on this train-test split ratio. We are asking for this input at this point because one of goals of the project is to better understand how this choice influences the outcomes of the model.

Comment (Bolaji61) [reply dev/Omobolaji/vehicles.ipynb]: Okay, I will. Thanks

Comment (Bolaji61) [reply dev/Omobolaji/vehicles.ipynb]: Yeah, I meant (15, 10). Deleted "10" accidentally. Thanks for your observation.

**Comment by Bolaji61** [comment]
**Train and Test a Classification Model on vehicles.csv dataset.**
I've been able to create a separate modules file containing the models and functions used. 
Next is to study the performance evaluation metrics and give detailed explanations of each line of code. 
@dzeber , I hope my pace is not too slow, I'm quite new to open source. Thanks for your earlier review.

**Comment by Bolaji61** [comment]
I have properly documented my thought process on my choice of model and test_size fraction. Next, is to tune the parameters of the models chosen in order to achieve a higher accuracy.

**Comment by dzeber** [comment]
Thanks for these updates - this has developed into a nicely presented, well-documented notebook. I'm merging this now, as it satisfies the requirements for #2. If you wish to continue working on it, I recommend checking out the other issues in the repo.

A couple of general comments (don't need to fix at present): 

- The `_model()` functions in your modules.py have a lot of repeated code. An improvement would be to combine into a single function and supply the parts that change as params.
- For the presentation of your very nice analysis of test split sizes, rather than hard-coding the results to present in the table, why not run the evaluations in the notebook, capture them dynamically and present the results directly.

**Comment by Bolaji61** [comment]
Yaay, this is great news. Thanks a lot, its my first ever merged PR in an Open Source project. I'll work on other issues in the project and fix the issues you commented on later as suggested. 
I do appreciate your reviews a lot, I hope to help some others this same way when I'm advanced. Do have a great weekend.

**Comment by Bolaji61** [comment]
> Thanks for these updates - this has developed into a nicely presented, well-documented notebook. I'm merging this now, as it satisfies the requirements for #2. If you wish to continue working on it, I recommend checking out the other issues in the repo.
> 
> A couple of general comments (don't need to fix at present):
> 
>     * The `_model()` functions in your modules.py have a lot of repeated code. An improvement would be to combine into a single function and supply the parts that change as params.
> 
>     * For the presentation of your very nice analysis of test split sizes, rather than hard-coding the results to present in the table, why not run the evaluations in the notebook, capture them dynamically and present the results directly.

The first suggestion has been fixed accordingly
